### PR TITLE
DBZ-2547 Include libaio for Oracle Instant Client compatibility

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-2547

### CODE REVIEW

**Comment by jpechane** [comment]
@Naros I am confused a little bit? Is this needed for Kafka broker?

**Comment by Naros** [comment]
@jpechane Oh I had thought I had put it on Connect but I guess I didn't.  I'll adjust.

File: connect-base/1.6/Dockerfile [hunk 1]
```diff
@@ -2,6 +2,9 @@ FROM debezium/kafka:1.6
 
 LABEL maintainer="Debezium Community"
 
+USER root
+RUN yum -y install libaio && yum clean all
+
```
Comment (jpechane) [inline connect-base/1.6/Dockerfile:7]: You should switch back to the user `kafka`

**Comment by jpechane** [comment]
@Naros Applied, thanks

**Comment by BunnyMan1** [comment]
Hi @Naros. Not sure if this is the right place to describe my issue but I think due to this change, I'm getting an error:
```
Building connect
Sending build context to Docker daemon  240.3MB
Step 1/10 : ARG DEBEZIUM_VERSION
Step 2/10 : FROM debezium/connect:$DEBEZIUM_VERSION
 ---> ed7fb1abe504
Step 3/10 : ENV KAFKA_CONNECT_JDBC_DIR=$KAFKA_CONNECT_PLUGINS_DIR/kafka-connect-jdbc
 ---> Using cache
 ---> 272ff0a91837
Step 4/10 : ENV INSTANT_CLIENT_DIR=/instant_client/
 ---> Using cache
 ---> 6233e263363d
Step 5/10 : USER root
 ---> Using cache
 ---> 4f87482dcb43
Step 6/10 : RUN yum -y install libaio && yum clean all
 ---> Running in 896120915e48
**/bin/sh: line 1: yum: command not found
The command '/bin/sh -c yum -y install libaio && yum clean all' returned a non-zero code: 127
ERROR: Service 'connect' failed to build : Build failed**
```

I'm using Oracle Connector Docker Image and this error shows up from v1.7 and v1.8. 
Image Build happens without issue on v1.6 and below but I'm looking to upgrade the version due to other issues that have been fixed since v1.7.

OS: Ubuntu 20.04.3 LTS.

Let me know if you need this to be created as an issue on Jira.

**Pull request merged**